src/consts/routePaths.ts

```diff
@@ -0,0 +1,6 @@
+export enum ROUTE_PATHS {
+  PRODUCTS = '/products',
+  ORDERS = '/orders',
+  SIGNUP = '/sign-up',
+  SIGNIN = '/sign-in',
+}
```

Comment:
I would suggest to use enums as follows (based on [TS docs](https://www.typescriptlang.org/docs/handbook/enums.html)):
```suggestion
export enum RoutePath {
  Products = '/products',
  Orders = '/orders',
  SignUp = '/sign-up',
  SignIn = '/sign-in',
}
```
Enum name and its keys in PascalCase. Enum name in the singular.
File name I would also change as the name of enum.